src/particles.cpp

```diff
@@ -58,7 +58,13 @@ Particle::Particle(
 	bool vertical,
 	video::ITexture *texture,
 	v2f texpos,
-	v2f texsize
+	v2f texsize,
+	u32 material_type_param,
+	u16 vertical_frame_num,
+	u16 first_frame,
+	float frame_length,
+	bool loop_animation,
+	const video::SColor glow
```

Comment:
@sfan5 est31 ask me to add const on passing this variable at server side and I mindlessly add const keyword at client side as well. My bad. 
Why we all have numbers? I wonder if there is Foghrye3 somewhere in parallel universe...